debugger/ios-list-usb-devices/src/Program.cs

```diff
@@ -19,21 +18,34 @@ private static int Main(string[] args)
 
             InitialiseWinSock();
 
-            var thread = new Thread(ThreadFunc);
-            thread.Start(args);
+            var iosSupportPath = args[0];
+            var pollingInterval = TimeSpan.FromMilliseconds(int.Parse(args[1]));
+
+            var pollingCancellationSource = new CancellationTokenSource();
+            var readLineTask = CreateAwaitStopCommandTask();
+            var innerPollingTask = CreatePollForDevicesTask(iosSupportPath, pollingInterval,
```

Comment:
var innerPollingTask = await CreatePollForDevicesTask(iosSupportPath, pollingInterval,
                pollingCancellationSource.Token);

and no need to await below.

Reply:
I need to await both the read from the command line and the poll at the same time. If the read comes in, it needs to cancel the poll. Without the `ContinueWith`, the final await will throw an unhandled `OperationCancelledException`, and any exception thrown by the poll is unobserved and doesn't get reported

Reply:
got it, thanks